### DISCUSSION TITLE
VirusTotal detection?

### DISCUSSION THREAD

**Discussion by adamrgolf** [Tyrrrz/YoutubeDownloader#297] [Q&A]
I checked YoutubeDownloader against VirusTotal and it came back with one result: 

"BitDefenderTheta
Gen:NN.ZemsilCO.34784.fm0@aCZ2M7m"

Can anyone help confirm what this is? Thank you!

**Comment by ScottsplaysSus** [comment]
Probably a false-positive, I've used the app for a long time and never had any issues related to virus. To be safe, just wait for the owner to reply.

**Reply by Tyrrrz** [reply]
Yeah it's probably the bootstrapper triggering those false-positives. If you want, you can compile from source in debug mode and it shouldn't raise any virus warnings.
